closed/autoconf/custom-hook.m4

```diff
@@ -164,8 +164,18 @@ AC_DEFUN_ONCE([OPENJDK_VERSION_DETAILS],
 [
   OPENJDK_SHA=`git -C $TOPDIR rev-parse --short HEAD`
   OPENJDK_TAG=`git -C $TOPDIR describe --abbrev=0 --tags --match "jdk-10*" "${OPENJDK_SHA}"`
+
+  if test "x$OPENJDK_TAG" = x; then
+    LAST_TAGGED_REVISION=`git -C $TOPDIR rev-list --tags --max-count=1`
+    OPENJDK_TAG=`git -C $TOPDIR describe --abbrev=0 --tags --match "jdk-10*" "${LAST_TAGGED_REVISION}"`
```

Comment:
This should not have been added for earlier versions: it was a hack to workaround an issue with the build flow at AdoptOpenJDK. If they clone the repo (at least enough to include tags) the assignment to `OPENJDK_TAG` above will work.